include/config/battle.h

```diff
@@ -192,6 +192,8 @@
 #define B_CRITICAL_CAPTURE          TRUE       // If set to TRUE, Critical Capture will be enabled.
 #define B_LAST_USED_BALL            TRUE       // If TRUE, the "last used ball" feature from Gen 7 will be implemented
 #define B_LAST_USED_BALL_BUTTON     R_BUTTON   // If last used ball is implemented, this button (or button combo) will trigger throwing the last used ball.
+#define B_LAST_USED_BALL_CYCLE      FALSE      // If TRUE, then holding B_LAST_USED_BALL_BUTTON while pressing the D-Pad cycles through the balls
+#define B_LAST_USED_BTN_CANCEL_CYC  B_BUTTON   // If last used ball cycling is implemented and the B_LAST_USED_BALL_BUTTON is held, but not yet let go, this button (or button combo) will cause no ball to be thrown when letting go of the button.
```

Comment:
> I played around with this and I think it would be more natural to have the cycling automatically cancel the "throw ball when letting go", since I think moving to an entire separate button is a bit non-intuitive.

Can you elaborate? Do you mean that you'd use R to cycle and when you let go, you stop cycling? And only when you push R (or whatever button you map without cycling, then you'll throw the ball?

Reply:
> Can you elaborate? Do you mean that you'd use R to cycle and when you let go, you stop cycling?

Well, no, I think the R + DPad for cycling is good :)
> only when you push R (or whatever button you map without cycling, then you'll throw the ball?

This is more what I was talking about :)

Reply:
Roger that. Change made to fit that logic.